content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -12,12 +12,26 @@ export class _StartupOverlay extends React.PureComponent {
     this.initScene = this.initScene.bind(this);
     this.removeOverlay = this.removeOverlay.bind(this);
     this.onInputInvalid = this.onInputInvalid.bind(this);
+    this.setFormFields = this.setFormFields.bind(this);
 
-    this.state = {emailInput: ""};
+    this.state = {emailInput: "", flowId: "", flowBeginTime: ""};
     this.initScene();
   }
 
   initScene() {
+    let req = new XMLHttpRequest();
+    req.onreadystatechange = () => {
+      if (req.readyState === 4 && req.status === 200) {
+        this.setFormFields(req.responseText);
+      }
+    };
+    req.onerror = e => {
+      console.error("fxa error", e);
+    };
+
+    req.open("GET", "https://accounts.firefox.com/metrics-flow");
```

Comment:
can we use any existing FxA settings or setup a setting in activity stream that can be changed in `about:config` to alter this url?

Reply:
I found identity.fxaccounts.remote.root, but it sounds like you'd want to be able to change the whole thing.
I'll make a new one, how about identity.fxaccounts.remote.metrics?
(I also noticed there are urls in about:config for legal/terms and legal/privacy - I'll add those to a different PR)

Reply:
though perhaps adding a new pref for this url should also go in a separate PR since this one will need to be uplifted

Reply:
I take that back, I'll use a pref and fallback to the hardcoded url if there is no pref, then if the pref code doesn't get uplifted this will at least still work, and then enterprise folks or others who wish to block this could.
sound okay?